src/app/searchResults/searchResults.tpl.html

```diff
@@ -14,7 +14,7 @@ <h1 ng-if="!$ctrl.searchParams.type || $ctrl.searchParams.type === 'people'">
 
       <div class="row">
         <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-          <form method="get" action="/search-results.html">
+          <form method="get" action="">
```

Comment:
Yes, keeps the current url.  This way it will work if it's search-results.html or search.html.  I could remove it, empty is the same as nothing.